governance.md

```diff
@@ -60,15 +61,12 @@ responsible for the operation of the SIG and for communication and
 coordination with the other SIGs, the Steering Committee, and the
 broader community.
 
-We still have work to do to unify people organization via SIGs and
-code organization using [OWNERS](contributors/devel/owners.md), and to
-find homes for all subparts of the project, such as the build
-system. We also need to create a charter for each SIG to more clearly
-specify its scope (topics, subsystems, code repos and directories),
+Each SIG must havea charter that
+specifies its scope (topics, subsystems, code repos and directories),
 responsibilities, areas of authority, how members and roles of
 authority/leadership are selected/granted, how decisions are made, and
-how conflicts are resolved. A template for intra-SIG governance should
-be developed in order to simplify SIG creation, but SIGs should be
+how conflicts are resolved. A template for intra-SIG governance is
+being developed in order to simplify SIG creation, but SIGs should be
```

Comment:
Should link the merged short template, and say additional templates are being developed?